qiskit_dynamics/backend/dynamics_backend.py

```diff
@@ -546,14 +550,28 @@ def _validate_run_input(run_input, accept_list=True):
         raise QiskitError(f"Input type {type(run_input)} not supported by DynamicsBackend.run.")
 
 
-def _get_acquire_data(schedules, valid_subsystem_labels):
+def _get_acquire_instruction_timings(
+    schedules: List[Schedule], valid_subsystem_labels: List[int], dt: float
+) -> Tuple[List[List[float]], List[List[int]], List[List[int]]]:
     """Get the required data from the acquire commands in each schedule.
 
-    Additionally validates that each schedule has acquire instructions occurring at one time,
-    at least one memory slot is being listed, and all measured subsystems exist in
-    subsystem_labels.
+    Additionally validates that each schedule has acquire instructions occurring at one time, at
+    least one memory slot is being listed, and all measured subsystems exist in
+    ``valid_subsystem_labels``.
+
+    Args:
+        schedules: A list of ``Schedule`` instances. valid_subsystem_labels: Valid acquire channel
+        indices. dt: The sample size.
+    Returns:
+        A Tuple of Lists containing, for each schedule: the list of integration intervals required
+        for each schedule (in absolute time, from 0.0 to the beginning of the acquire instructions),
+        a list of the subsystems being measured, and a list of the memory slots indices in which to
+        store the results of each subsystem measurement.
+    Raises:
+        QiskitError: If a schedule contains no measurement, if a schedule contains measurements at
+            different times, or if a measurement has an invalid subsystem label.
     """
-    t_span_list = []
+    acquire_time_list = []
```

Comment:
```suggestion
    t_span_list = []
```

Reply:
oops, accidentally missed this one, but fixed it in the subsequent commit with linting.